cmd/mdatagen/main.go

```diff
@@ -62,22 +62,22 @@ func run(ymlPath string) error {
 				filepath.Join(codeDir, "generated_status.go"), md, "metadata"); err != nil {
 				return err
 			}
-			if err = generateFile(filepath.Join(tmplDir, "telemetry.go.tmpl"),
```

Comment:
Why move these few lines?

Reply:
Not super clear from the description, but the commit for the change describes it:

> ensure package test and telemetry is generated for pkg type